tests/qgis-projects/tests/test_filter_layer_data_by_polygon_for_groups.qgs

```diff
@@ -0,0 +1,1310 @@
+<qgis projectname="" saveDateTime="2021-06-22T17:54:09" saveUser="mdouchin" saveUserFull="mdouchin" version="3.16.5-Hannover">
```

Comment:
The project needs to be updated according to what was done in https://github.com/3liz/lizmap-plugin/pull/355